crates/frontend/src/pages/context_override.rs

```diff
@@ -151,7 +151,25 @@ pub fn context_override() -> impl IntoView {
 
     let handle_context_create = Callback::new(move |_| {
         set_form_mode.set(Some(FormMode::Create));
-        set_selected_data.set(None);
+        let PageResource {
```

Comment:
you can write it like this as well:
```suggestion
        let PageResource { dimensions, .. } = page_resource.get().unwrap_or_default();
```

you wont need to write the other fields and then ignore them explicitly by using underscore